android/app/webviewAssets.gradle

```diff
@@ -0,0 +1,33 @@
+/**
+ * Build and merge static webview assets.
+ *
+ * This is largely the job of a script external to Gradle, as there is overlap
+ * between the iOS and Android builds.
+ */
+
+gradle.projectsEvaluated {
+    // The root of our git repository.
+    def repoDir = project.file('../..').absolutePath
+
+    android.applicationVariants.all { variant ->
+        // Accurate as of Android Gradle Plugin 3.4.0. Known to move between
+        // releases; but alas, no stable API is known to the Internet at large.
+        def assetsDir = "${buildDir}/intermediates/merged_assets/${variant.name}/out"
```

Comment:
Alas indeed.

I think a helpful breadcrumb to add here, too, would be something like `See commits 8f8a266af and 833d083a9 for previous changes.`

Reply:
Good thought -- thanks!